src/util/NamedColor.cpp

```diff
@@ -0,0 +1,89 @@
+#include "NamedColor.h"
+
+#include <sstream>
+
+#include <util/StringUtils.h>
```

Comment:
```suggestion
#include "util/StringUtils.h"
```

Reply:
Yep, that's a stupid mistake :sweat_smile: